webapp/public/js/domjudge.js

```diff
@@ -597,8 +594,8 @@ function updateMenuInternalErrors(data)
         for(var i=0; i<num; i++) {
             sendNotification('Judgehost internal error occurred.',
                 {'tag': 'ie_'+data[i].errorid,
-                 'link': domjudge_base_url + '/internal-errors/' + data[i].errorid,
-                 'body': data[i].description});
+                    'link': domjudge_base_url + '/internal-errors/' + data[i].errorid,
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mvr320/domjudge/pullRequest?prid=8427924)
```suggestion
                 "link": domjudge_base_url + '/internal-errors/' + data[i].errorid,
```